common/volume_definitions.go

```diff
@@ -93,4 +95,24 @@ var (
 		MountPath:        udevPath,
 		MountPropagation: &hostContainerPropagation,
 	}
+
+	// DiscoveryMetricsCertVolume is the corev1.Volume definition for serving cert secret for discovery service
+	DiscoveryMetricsCertVolume = corev1.Volume{
+		Name: metricsServingCert,
+		VolumeSource: corev1.VolumeSource{
+			Secret: &corev1.SecretVolumeSource{
+				SecretName: DiscoveryMetricsServingCert,
+			},
+		},
+	}
+
+	// DiskmakerMetricsCertVolume is the corev1.Volume definition for serving cert secret for diskmaker service
+	DiskmakerMetricsCertVolume = corev1.Volume{
+		Name: metricsServingCert,
+		VolumeSource: corev1.VolumeSource{
+			Secret: &corev1.SecretVolumeSource{
+				SecretName: DiskMakerMetricsServingCert,
+			},
+		},
+	}
```

Comment:
same as above. This seems like pretty static information and can be moved to yaml directly.

Reply:
same reason as mentioned [here](https://github.com/openshift/local-storage-operator/pull/249#discussion_r673306183). Let me know your views.